index.test-d.ts

```diff
@@ -0,0 +1,4 @@
+import { expectType } from 'tsd';
+import remoteGitTags = require('.');
+
+expectType<(url: string) => Promise<Map<string, string>>>(remoteGitTags);
```

Comment:
Test the return type of calling the function, not the whole function since that's just duplicating what's in `index.d.ts`.